sources/packages/web/src/components/institutions/request-a-change/AvailableToChangeSummary.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <tab-container>
+    <body-header-container>
+      <template #header>
+        <body-header title="Applications" :recordsCount="applications.count">
+          <template #subtitle>
+            Request a change for a program and offering in an application
+            <tooltip-icon
+              >Only "completed" applications can be requested to change. For
+              applications that are in progress, please have the student edit
+              their application from their account. If the student has received
+              funding already, the student will need to "request a change" from
+              their account.</tooltip-icon
+            >
+          </template>
+          <template #actions>
+            <v-text-field
+              density="compact"
+              label="Search name or application #"
+              variant="outlined"
+              v-model="searchCriteria"
+              data-cy="searchCriteria"
+              @keyup.enter="searchApplicationOfferingChangeRecords"
+              prepend-inner-icon="mdi-magnify"
+              hide-details="auto"
+            >
+            </v-text-field>
+          </template>
+        </body-header>
+        <content-group>
+          <toggle-content :toggled="!applications.count">
+            <v-data-table-server
+              :headers="AvailableToChangeOfferingChangeSummaryHeaders"
+              :items="applications.results"
+              :items-length="applications.count"
+              :loading="loading"
+              :items-per-page="DEFAULT_PAGE_LIMIT"
+              @update:options="paginationAndSortEvent"
+            >
+              <template #[`item.fullName`]="{ item }">
+                <span data-cy="fullName">{{ item.columns.fullName }} </span>
```

Comment:
Just to clarify. Are we still considering adding data-cy?

Reply:
so, there wn;t be any automation test happing anything? I don't know the current status, but  I was under the impression that there will be automation testing happing and we decided to add data_cy for future developments, I don't remember any discussion mentioning we need to stop it. @andrewsignori-aot  @JasonCTang @guru-aot @andrepestana-aot

Reply:
I prefer to avoid using it at all for now since we have no directions.
If we have the commitment to continue to add it we also need to be clear about how we will be adding.

Reply:
okay, then removing the `data-cy`